linera-views/src/dynamo_db.rs

```diff
@@ -1185,6 +1236,14 @@ pub enum DynamoDbContextError {
     #[error("The key must be of strictly positive length")]
     ZeroLengthKey,
 
+    /// The key must have at most 1024 bytes
+    #[error("The key must have at most 1024 bytes")]
+    TooLongKey,
+
+    /// The key prefix must have at most 1024 bytes
+    #[error("The key prefix must have at most 1024 bytes")]
+    TooLongKeyPrefix,
```

Comment:
idem (you had it right with `ValueLengthTooLarge` below!)

Reply:
Done.